src/bscPlugin/transpile/BrsFilePreTranspileProcessor.spec.ts

```diff
@@ -3,6 +3,8 @@ import * as fsExtra from 'fs-extra';
 import { Program } from '../../Program';
 import { standardizePath as s } from '../../util';
 import { tempDir, rootDir } from '../../testHelpers.spec';
+import { Logger, LogLevel } from '../..';
```

Comment:
Can you change this to point to the actual logger file? Sometimes importing `../..` causes circular references.

Reply:
Fixed!